modules/integration/src/test/scala/scala/cli/integration/InstallAndUninstallCompletionsTests.scala

```diff
@@ -0,0 +1,49 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+class InstallAndUninstallCompletionsTests extends munit.FunSuite {
+  val zshRcFile  = ".zshrc"
+  val bashRcFile = ".bashrc"
+  val rcContent = s"""
+                      |dummy line
+                      |dummy line"""
+  val testInputs = TestInputs(
+    Seq(
+      os.rel / zshRcFile  -> rcContent,
+      os.rel / bashRcFile -> rcContent
+    )
+  )
+
+  def runInstallAndUninstallCompletions(): Unit = {
+    testInputs.fromRoot { root =>
+      val zshRcPath  = root / zshRcFile
+      val bashRcPath = root / bashRcFile
+      // install completions to the dummy rc files
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", zshRcPath, "--shell", "zsh").call(
+        cwd = root
+      )
+      os.proc(TestUtil.cli, "install-completions", "--rc-file", bashRcPath, "--shell", "bash").call(
+        cwd = root
+      )
+      val completionsStart = s"# >>> scala-cli completions >>>"
+      val completionsEnd   = s"# <<< scala-cli completions <<<"
+      expect(!os.read.lines(zshRcPath).filter(_ == completionsStart).isEmpty)
+      expect(!os.read.lines(zshRcPath).filter(_ == completionsEnd).isEmpty)
+      expect(!os.read.lines(bashRcPath).filter(_ == completionsStart).isEmpty)
+      expect(!os.read.lines(bashRcPath).filter(_ == completionsEnd).isEmpty)
```

Comment:
NIT Slightly simpler:
```suggestion
      val zshRcLines = os.read.lines(zshRcPath)
      val bashRcLines = os.read.lines(bashRcPath)
      expect(zshRcLines.contains(completionsStart))
      expect(zshRcLines.contains(completionsEnd))
      expect(bashRcLines.contains(completionsStart))
      expect(bashRcLines.contains(completionsEnd))
```